src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform $global-transition-time ease;
```

Comment:
The transition property should be applied to the specific property you want to animate, not the whole element. In this case, it should be applied to the 'transform' property.